docs/install/installEverest.md

```diff
@@ -0,0 +1,116 @@
+# Install Everest
+
+## Setup
+
+Percona Everest has two primary components:
+
+- [Percona CLI](https://github.com/percona/percona-everest-cli), which connects Percona Everest to your Kubernetes cluster.
+- Everest application (UI for simplicity), which you can spin up using the instructions below.
+
+### Supported operators
+
+- Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.13.0
+- Percona Operator for MongoDB (PSMDB) 1.14.0
+- Percona Operator for PostgreSQL (PG) 2.2.0
+
+## Before you start
+
+Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a Kubernetes cluster.
+Therefore, before installing Everest, ensure that you have a publicly accessible Kubernetes cluster.
+
+We recommend [setting one up on the Amazon Elastic Kubernetes Service (EKS)](../install/eks.md) or on [Google Kubernetes Engine (GKE)](../install/GKE-k8s-cluster), as it may be difficult to currently spin one up on local Kubernetes installations (minikube, kind, k3d, or similar products).
+
+## Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+
+1. [Install Docker Desktop](https://docs.docker.com/desktop), which also includes Docker Engine and the Docker compose plugin.
+2. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install) for GKE.
+3. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under *~/.kube/config*. Run the following command:
+    ```sh 
+    kubectl get nodes`
+    ```
+
+    ??? example "Expected output"
+        ```
+            NAME                                       STATUS   ROLES    AGE   VERSION
+             gke-<name>-default-pool-75d48bfc-bx8g   Ready    <none>   11h   v1.26.7-gke.500
+             gke-<name>-default-pool-75d48bfc-c2df   Ready    <none>   11h   v1.26.7-gke.500
+             gke-<name>-default-pool-75d48bfc-zl7k   Ready    <none>   11h   v1.26.7-gke.500
+         ```
+
+## Get started
+
+To install and provision Percona Everest:
+
+1. Download the Docker compose file:
+```
+curl https://raw.githubusercontent.com/percona/percona-everest-backend/release-0.3.0/deploy/quickstart-compose.yml -o quickstart-compose.yml
+```
+2. Initialize the Everest container and its internal PostgreSQL database by executing one of the following commands, each offering distinct access configurations:
```

Comment:
Why is there only PostgreSQL? When running quick install script, I get all Operators deployed. How to deploy others following these steps?

Reply:
@nastena1606 I think it is about internal PostgreSQL DB, which Everest uses. It is not about operators at all.